seed.js

```diff
@@ -0,0 +1,134 @@
+const db = require('./database/index.js');
+const Promise = require('bluebird');
+const faker = require('faker');
+
+//create an array of random size, with random user_id's
+var randomUsers = () => {
+  var arr = new Array(Math.floor(Math.random() * 15));
+
+  for (var i = 0; i < arr.length; i++) {
+    arr[i] = Math.floor(Math.random() * 99 + 1);
+  }
+  return arr.join(' ').toString();
+}
+
+
+
+var createUsers = () => {
+  var users = [];
+  return new Promise((resolve, reject) => {
+
+    for ( let i = 1; i < 101; i++) {
+      var name = faker.name.firstName() + ' ' + faker.name.lastName()
+      const user = {
+        id: i,
+        name: name,
+        username: name.split(' ').join('.'),
+        address: faker.address.city() + ", " + faker.address.country(),
+        contributions: Math.floor(Math.random() * 500),
+        votes: Math.floor(Math.random() * 500),
+        avatar: faker.internet.avatar(),
+        followers: Math.floor(Math.random() * 2000)
+      }
+      users.push(user);
+    }
+    resolve(users)
+
+  })
+
+
+}
+
+
+var addUsers= (users) => {
+
+    return new Promise((resolve, reject) => {
+      for (var i = 0; i < users.length-1; i++) {
```

Comment:
one thing to watch out for, I would maybe use let instead of var in the for loops, or use different letters instead of i, since var is not block scoped, not sure if that will cause potential errors